workflows/phylorthology.nf

```diff
@@ -48,66 +70,416 @@ include { INPUT_CHECK } from '../subworkflows/local/input_check'
 //
 // MODULE: Installed directly from nf-core/modules
 //
-include { FASTQC                      } from '../modules/nf-core/fastqc/main'
-include { MULTIQC                     } from '../modules/nf-core/multiqc/main'
-include { CUSTOM_DUMPSOFTWAREVERSIONS } from '../modules/nf-core/custom/dumpsoftwareversions/main'
-
+include { BUSCO as BUSCO_SHALLOW                } from '../modules/nf-core/busco/main'
+include { BUSCO as BUSCO_BROAD                  } from '../modules/nf-core/busco/main'
+include { BUSCO                                 } from '../modules/nf-core/busco/main'
+include { DIAMOND_BLASTP as DIAMOND_BLASTP_TEST } from '../modules/nf-core/diamond/blastp/main'
+include { DIAMOND_BLASTP                        } from '../modules/nf-core/diamond/blastp/main'
+include { MAFFT                                 } from '../modules/nf-core/mafft/main'
+include { MAFFT as MAFFT_REMAINING              } from '../modules/nf-core/mafft/main'
+include { IQTREE                                } from '../modules/nf-core/iqtree/main'
+include { IQTREE as IQTREE_REMAINING            } from '../modules/nf-core/iqtree/main'
 /*
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     RUN MAIN WORKFLOW
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
+//def meta = ch_spp
+// This could be put into either the config file or specified via commandline. 
 
-// Info required for completion email and summary
-def multiqc_report = []
+// Function to get list of [ meta, [ file ] ]
+def create_og_channel(LinkedHashMap row) {
+    // create meta map
+    def meta  = [:]
+        meta.og   = row.orthogroup
+        meta.nspp = row.num_spp
+        meta.copy_num = row.mean_copy_num
```

Comment:
These ones are produced in an orthogroup summary file produced by the "og-tax-summary.R" script called (currently) within the orthofinder_mcl.nf module, but as mentioned in a previous comment, should be broken out into its own module.

Reply:
The reason for making this second meta map is because we switch from having to parallelize/distribute tasks across species (sequence processing, blasting, etc), to across orthogroups (MSAs, gene family tree inference, etc).